pkg/render/node.go

```diff
@@ -836,6 +836,8 @@ func (c *nodeComponent) nodeVolumes() []corev1.Volume {
 			corev1.Volume{Name: "sys-fs", VolumeSource: corev1.VolumeSource{HostPath: &corev1.HostPathVolumeSource{Path: "/sys/fs", Type: &dirOrCreate}}},
 			// Volume for the bpffs itself, used by the main node container.
 			corev1.Volume{Name: "bpffs", VolumeSource: corev1.VolumeSource{HostPath: &corev1.HostPathVolumeSource{Path: "/sys/fs/bpf", Type: &dirMustExist}}},
+			// Volume used by mount-cgroupv2 init container to access root cgroup name space of node.
+			corev1.Volume{Name: "proc", VolumeSource: corev1.VolumeSource{HostPath: &corev1.HostPathVolumeSource{Path: "/proc"}}},
```

Comment:
It looks like the code only uses `/node-proc/1/ns`, could this volume be reduced in scope to `/proc/1/ns`?

Reply:
Worth a shot!  `/proc/1/ns` is still "very privileged" but might as well reduce our scope as much as possible. I don't think PID 1 can go away or the system panics so it may be the only process where that is workable.

Reply:
Fixed it.

Reply:
@tmjd now we need to also access `/proc/1/mountinfo`, so I limited the mount to `/proc/1`. How does it sound?